### PR TITLE
Feature/contact api

### PR DESCRIPTION
**Pull request by tedralin**

### CODE REVIEW

**Review by tedralin — COMMENTED** [review]
Remove delete of: CurrencyController, CurrencyNamesController, CurrencyRateController, FXController; FXCodesNames, FXRateFullAPI,  CurrencyNamesService, FXRateServiceGet these from Master
Get UserController from Master

File: src/main/java/com/nology/synergyapi/controller/UserController.java [hunk 1]
```diff
@@ -1,6 +1,6 @@
 package com.nology.synergyapi.controller;
 
-import com.nology.synergyapi.data.Repository.UserRepository;
```
Comment (shushpan-v-satori) [inline src/main/java/com/nology/synergyapi/controller/UserController.java:3]: Repository shall be under /data folder

File: src/main/resources/application.properties [hunk 2]
```diff
@@ -3,5 +3,7 @@ spring.jpa.hibernate.naming.physical-strategy = org.hibernate.boot.model.naming.
 spring.jpa.hibernate.naming.implicit-strategy = org.hibernate.boot.model.naming.ImplicitNamingStrategyLegacyJpaImpl
 spring.datasource.url=jdbc:mysql://${MYSQL_HOST:localhost}:3306/synergy
 spring.datasource.username=root
-spring.datasource.password=nology!
+spring.datasource.password=mysqlpwd
```
Comment (shushpan-v-satori) [inline src/main/resources/application.properties:6]: please put it to gitignore

**Review by shushpan-v-satori — APPROVED** [review]

File: src/main/java/com/nology/synergyapi/controller/UserController.java (hunk 1 above)
Comment (tedralin) [reply src/main/java/com/nology/synergyapi/controller/UserController.java:3]: repository in the same level as data

File: src/main/resources/application.properties (hunk 2 above)
Comment (tedralin) [reply src/main/resources/application.properties:6]: add

**Review by shushpan-v-satori — APPROVED** [review]

**Pull request merged**